gitpod-io/website
# fix: not required `h` tags to `div` tags

> fixes #3063

---

## Description

- Alt tag for Indicator
- Fixed header tags on homepage:
    - [Technical] components - `get- started/project`, `choose-project`.
    - [Visual/Text]: Following text part got changed from headers to div with same styles. 
        - `Select project, check dependencies, checkout branch, viewreadme.txt, install tools, run build, run test, check dependencies,checkout branch, viewreadme.txt, install tools,run build, run test, start...`
        - `Node or TypeScript` 
        - `Python`
        - ` Golang`
        - ` Rust`
        - `Java`
        - `Svelte`

## How to Test

- Open preview
- :eyes: see the changes on homepage (`/`) described above^



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3176"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
